File: Data Structures & Algorithms/two-integer-sum/submission-0.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int[] indices = new int[2];
        HashMap<Integer, Integer> map = new HashMap<>();

        for(int i = 0; i < nums.length; i++) {
            int comp = target - nums[i];
            if(map.containsKey(comp)) {
                indices[0] = map.get(comp); 
                indices[1] = i;
                break; // We found the indices, no need to continue
            } else {
                map.put(nums[i], i);
            }
        }
        return indices;
    }
}

File: Data Structures & Algorithms/majority-element-ii/submission-0.py
class Solution:
    def majorityElement(self, nums: List[int]) -> List[int]:

        count = {}
        res = []

        for n in nums:
            count[n] = count.get(n, 0) + 1

        for key in count:
            if count[key] > len(nums) / 3:
                res.append(key)
        
        return res
        